middleware/2fa.js

```diff
@@ -219,7 +235,7 @@ const initCode = async (ctx) => {
         return;
     }
     // client waits to deploy contract until code is verified
-    await sendCode(ctx, method, twoFactorMethod);
+    await sendCode(ctx, method, twoFactorMethod, -1, accountId);
```

Comment:
what is the reason behind this change?

Reply:
https://github.com/near/near-contract-helper/pull/363/commits/db96886a2ce2fa61e7f35b2ec4de5497410daf5f

Prior to this, subject line for 2fa email never included account Id even though it attempted to include it.  Emit of security code events includes account Id that the security code was generated for, which highlighted the problem beyond just the subject line being broken in 2fa emails :)